doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -33,7 +33,9 @@ The following sections provide detailed lists of changes by component.
 MCUboot
 =======
 
-|no_changes_yet_note|
+* Added:
+
+  * Option to prevent inclusion of default nRF5340 network core DFU image hook, which allows a custom implementation by users if :kconfig:option:`CONFIG_BOOT_IMAGE_ACCESS_HOOK_NRF5340` is disabled (enabled by default). CMake can be used to add additional hook files, see :file:`modules/mcuboot/hooks/CMakeLists.txt` for an example of how to achieve this.
```

Comment:
```suggestion
  * An option to prevent inclusion of default nRF5340 network core DFU image hook, which allows a custom implementation by users if the :kconfig:option:`CONFIG_BOOT_IMAGE_ACCESS_HOOK_NRF5340` Kconfig option is disabled (enabled by default).
  CMake can be used to add additional hook files.
  See :file:`modules/mcuboot/hooks/CMakeLists.txt` for an example of how to achieve this.
```

Reply:
Fixed